tests/Command/RunCommandTest.php

```diff
@@ -1090,17 +1067,17 @@ public function testExecuteStateTransitionError(): void
             'command' => $command->getName(),
         ]);
 
+        self::assertFalse($testHandler->hasErrorRecords());
+        self::assertFalse($testHandler->hasCriticalRecords());
+        self::assertFalse($testHandler->hasWarningRecords());
+
+        self::assertTrue($testHandler->hasInfoThatContains(
+            'Running job "123".'
+        ));
         self::assertTrue($testHandler->hasNoticeThatContains(
             'Failed to save result for job "123". State transition forbidden:'
         ));
         self::assertEquals(0, $ret);
-
-        $events = $this->storageClient->listEvents(['runId' => '124']);
-        $messages = array_column($events, 'message');
```

Comment:
Tím že ten test měl fixní runId, tak je tenhle check k ničemu. Načítalo to eventy i z předchozích běhů. A že to loguje není potřeba testovat, stačí imho vzít ty záznamy z loggeru

Reply:
no to, je dobrej gól :)